src/classes/CW_BriefApprovalProcessTest.cls

```diff
@@ -0,0 +1,297 @@
+/**
+* File Name   :    CW_BriefApprovalProcessTest.cls
+* Description :    B-106804 As a Marketing Ops I want the ability to submit a MarCom Brief
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 24/04/2014 Cristina P.	Intial
+*
+*/
+@isTest
+private class CW_BriefApprovalProcessTest 
+{
+	public static User approverUser;
+	public static User nonApproverUser;
+	public static CW_Campaign__c camp;
+	
+	public static ApproverListSetting__c approverSetting;
+	
+	public static void initUsersAndCustomSettings()
+	{
+    	User thisUser = [SELECT Id FROM User WHERE Id = :UserInfo.getUserId()];
```

Comment:
Problem: Unbound SOQL query. Reason: a SOQL query can return up to 50,000 records. If this number is exceeded a runtime exception is thrown. Suggestion: cap the max number of records returned adding a LIMIT clause to the query.